server/paths/manage.ts

```diff
@@ -100,19 +98,15 @@ const v2DateChangesPath = v2BookingPath.path('date-changes')
 const v2ExtensionsPath = v2BookingPath.path('extensions')
 const outOfServiceBedsPath = v2SinglePremisesPath.path('beds/:bedId/out-of-service-beds')
 const outOfServiceBedPath = outOfServiceBedsPath.path(':id')
-const outOfServiceBedsIndexPath = v2ManagePath.path('/out-of-service-beds')
+const outOfServiceBedsIndexPath = v2ManagePath.path('out-of-service-beds')
 
 const v2Manage = {
```

Comment:
As all the old manage paths are going, could this just be `manage` rather than v2Manage in the path tree.

Reply:
Yes, in fact it could be squashed some more, but this may have to be done at a later point as long as we still have _some_ references to v2 in the codebase.

Reply:
In fact, this should really be done -- however we do need to retain historical `v1` paths, so they can be redirected. How about we move those to `paths.manageV1`, and move any current `paths.v2Manage` to the root of `paths`?